src/toast/toast.js

```diff
@@ -137,11 +139,25 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
 
   animateOut = (callback: () => mixed = () => {}) => {
     this.setState({isVisible: false});
+
+    let animationDuration =
+      (__BROWSER__ &&
+        this.bodyRef &&
+        this.bodyRef.current &&
+        Number(
+          // $FlowFixMe: Body is typed as `mixed` but actually is Element so this isn't a problem
+          window
+            .getComputedStyle(this.bodyRef.current)
+            .getPropertyValue('transition-duration')
+            .replace('s', ''),
+        ) * 1000) ||
+      200; // default backup just in case
```

Comment:
Should I remove the `|| 200` altogether? Also if not I just realised need to handle the zero duration case coz that would also be falsy, will send patch (if we're keeping it)